### ISSUE TITLE
Pharos::Kube::Stack spec does not pass locally

### ISSUE THREAD

**Issue by kke** [kontena/pharos-cluster#699]
```
Failures:

  1) Pharos::Kube::Stack for a trivial test stack labels resources with the correct label and annotation
     Failure/Error: annotations: { :'pharos.kontena.io/stack-checksum' => subject.checksum },

     NoMethodError:
       undefined method `checksum' for #<Pharos::Kube::Stack:0x00007fe6aa9929b8>
     # ./spec/pharos/kube_stack_spec.rb:33:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:121:in `block (2 levels) in <top (required)>'
     # ./spec/spec_helper.rb:111:in `block (2 levels) in <top (required)>'
```

The spec calls `subject.checksum`:
https://github.com/kontena/pharos-cluster/blob/master/spec/pharos/kube_stack_spec.rb#L33

Subject is an instance of `Pharos::Kube::Stack` which is inherited from `k8s-client`'s [`K8s::Stack`](https://github.com/kontena/k8s-client/blob/master/lib/k8s/stack.rb) . It does not define a `checksum` method.

I don't know why it does not fail on drone.

**Comment by jnummelin** [comment]
let's re-open if this resurfaces

**Comment by kke** [comment]
It was fixed in #725
